static/css/tokens.css

```
@@ -58,24 +58,24 @@
   --color-text: #0f172a;
   --color-text-muted: #475569;
   --color-border: #e2e8f0;
-  --color-primary: #02243f;
-  --color-accent: #2db830;
+  --color-primary: #1a1e27;
+  --color-accent: #d9534f;
   --color-chip: #f1f5f9;
   --color-chip-ring: #cbd5e1;
   color-scheme: light;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)